tests/DomainZones/DomainZonesTest.php

```diff
@@ -277,6 +277,30 @@ public function testAdminDomainZonesAddMXInvalid()
 		DomainZones::getLocal($admin_userdata, $data)->add();
 	}
 
+	public function testAdminDomainZonesAddCAA()
```

Comment:
if validation is added (structure and only `issue`, `issuewild` and `iodef` etc.) we could use some more tests also with false data etc. to verify the validation